docs/getting-started-guides/fedora/flannel_multi_node_cluster.md

```diff
@@ -0,0 +1,219 @@
+#**Kubernetes multiple nodes cluster with flannel on Fedora**
+
+* This document describes how to deploy kubernetes on multiple hosts to set up a multi-node cluster and networking with flannel. Follow fedora [getting started guide](fedora_manual_config.md) to setup 1 master (fed-master) and 2 or more nodes (minions). Make sure that all nodes (minions) have different names (fed-node1, fed-node2 and so on) and labels (fed-node1-label, fed-node2-label, and so on) to avoid any conflict. Also make sure that the kubernetes master host is running etcd, kube-controller-manager, kube-scheduler, and kube-apiserver services, and the nodes (minions) are running docker, kube-proxy and kubelet services. Now install flannel on kubernetes nodes (minions). flannel on each node configures an overlay network that docker uses. flannel runs on each node to setup a unique class-C container network.
+
+**Perform following commands on the kubernetes master** 
+
+* Configure flannel by creating a `flannel-config.json` in your current directory on fed-master. flannel provides udp and vxlan among other overlay networking backend options. In this guide, we choose kernel based vxlan backend. The contents of the json are:
+
+```
+{
+    "Network": "18.16.0.0/16",
+    "SubnetLen": 24,
+    "Backend": {
+        "Type": "vxlan",
+        "VNI": 1
+     }
+}
+```
+**NOTE:** Choose an IP range that is *NOT* part of the public IP address range.
+
+* Add the configuration to the etcd server on fed-master.
+
+```
+# curl -L http://fed-master:4001/v2/keys/coreos.com/network/config -XPUT --data-urlencode value@flannel-config.json
+```
+
+* The output should look like:
+
+```
+{"action":"set","node":{"key":"/coreos.com/network/config","value":"{\n    \"Network\": \"18.16.0.0/16\",\n    \"SubnetLen\": 24,\n    \"Backend\": {\n        \"Type\": \"vxlan\",\n        \"VNI\": 1\n     }\n}\n","modifiedIndex":3,"createdIndex":8}}
+```
+
+* Verify the key exists in the etcd server on fed-master.
+
+```
+# curl -L http://fed-master:4001/v2/keys/coreos.com/network/config
+{"action":"get","node":{"key":"/coreos.com/network/config","value":"{\n    \"Network\": \"18.16.0.0/16\",\n    \"SubnetLen\": 24,\n    \"Backend\": {\n        \"Type\": \"vxlan\",\n        \"VNI\": 1\n     }\n}\n","modifiedIndex":8,"createdIndex":8}}
+```
```

Comment:
@eparis I agree that could look cleaner but I am not sure how to do that. I tried something but nothing seems to work.